.github/workflows/nightly.yml

```diff
@@ -45,11 +46,11 @@ jobs:
       - name: Additionally upload Nightly APK to browserstack for testing
         continue-on-error: true # don't block anything by this upload failing (for now)
         run: |
-          ./gradlew assembleNightly $CI_GRADLE_ARG_PROPERTIES
+#          ./gradlew assembleNightly $CI_GRADLE_ARG_PROPERTIES
           curl -u "$BROWSERSTACK_USERNAME:$BROWSERSTACK_PASSWORD" -X POST "https://api-cloud.browserstack.com/app-automate/upload" -F "file=@app/build/outputs/apk/nightly/app-universal-nightly.apk" -F "custom_id=element-x-android-nightly"
         env:
-          ELEMENT_ANDROID_MAPTILER_API_KEY: ${{ secrets.MAPTILER_KEY }}
-          ELEMENT_ANDROID_MAPTILER_LIGHT_MAP_ID: ${{ secrets.MAPTILER_LIGHT_MAP_ID }}
-          ELEMENT_ANDROID_MAPTILER_DARK_MAP_ID: ${{ secrets.MAPTILER_DARK_MAP_ID }}
+#          ELEMENT_ANDROID_MAPTILER_API_KEY: ${{ secrets.MAPTILER_KEY }}
```

Comment:
Don't we need these also for apks?

Reply:
No, the APK is built by the previous step and the env var are set there (line 37).